keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -173,3 +214,40 @@ def get_config(self):
         }
         base_config = super().get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+    def _cutout_bounding_boxes(self, image, bounding_boxes, transformation):
+        top = tf.cast(transformation["top"], dtype=self.compute_dtype)
```

Comment:
This implementation is that of RandomCrop. The `transformation` returns `center_x, center_y, rectangle_height, rectangle_width`.
this needs to be updated.